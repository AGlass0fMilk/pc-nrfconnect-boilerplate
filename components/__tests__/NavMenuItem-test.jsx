import React from 'react';
import renderer from 'react-test-renderer';
import {
    decorateNavMenuItem,
} from '../NavMenuItem';

describe('NavMenuItem', () => {
    it('should implement component decorator', () => {
        expect(decorateNavMenuItem).toBeInstanceOf(Function);
    });
});

describe('decorateNavMenuItem', () => {
    it('should render component', () => {
        const NavMenuItem = decorateNavMenuItem('div');
        expect(renderer.create(
            <NavMenuItem />,
        )).toMatchSnapshot();
    });
});
