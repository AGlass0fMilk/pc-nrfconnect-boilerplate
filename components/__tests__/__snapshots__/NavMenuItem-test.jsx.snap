exports[`decorateNavMenuItem should render component 1`] = `<div />`;
